readthedocs/api/v2/proxied_urls.py

```diff
@@ -16,3 +17,8 @@
 ]
 
 urlpatterns = api_footer_urls
+
+if 'readthedocsext.embed' in settings.INSTALLED_APPS:
+    urlpatterns += [
+        url(r'embed/', include('readthedocsext.embed.urls'))
```

Comment:
Just a note that if we want to keep this working in the normal API we need two sets of URLs, ones that work with the normal API and other ones that make use of the auth backends.

See https://github.com/readthedocs/readthedocs.org/blob/track-total-results/readthedocs/api/v2/views/proxied.py

Reply:
This endpoint is already working under `readthedocs.org`. I don't think it will stop working if it's added here, right?

Reply:
If you are talking about corporate, I don't think we want to make this endpoint available under `readthedocs.com`

Reply:
If this isn't going to be exposed on readthedocs.com, then it should be fine.

Reply:
@humitos I thought the point of this code was to enable it on corporate?

Reply:
@ericholscher it's going to be on .com, but only on the proxied api (foo.rtd-hosted.com/_/api/v2/), not on `readthedocs.com/api/v2/`

Reply:
Makes sense 👍